tests/integration/helpers.py

```diff
@@ -138,10 +139,48 @@ async def wait_for_mongos_units_blocked(
     try:
         old_interval = (await ops_test.model.get_config())[hook_interval_key]
         await ops_test.model.set_config({hook_interval_key: "1m"})
-        for attempt in Retrying(
-            stop=stop_after_delay(timeout), wait=wait_fixed(1), reraise=True
-        ):
+        for attempt in Retrying(stop=stop_after_delay(timeout), wait=wait_fixed(1), reraise=True):
             with attempt:
                 await check_all_units_blocked_with_status(ops_test, db_app_name, status)
     finally:
         await ops_test.model.set_config({hook_interval_key: old_interval})
+
+
+def get_port_from_node_port(ops_test: OpsTest, node_port_name: str) -> None:
```

Comment:
Helpers for node port test validation

Lets try to carve out some time to add this to [data platform helpers pip](https://github.com/canonical/data-platform-helpers)